modules/cointrafficBidAdapter.md

```diff
@@ -0,0 +1,24 @@
+# Overview
+
+```
+Module Name: Cointraffic Bidder Adapter
+Module Type: Cointraffic Adapter
+Maintainer: tech@cointraffic.io
+```
+
+# Description
+The Cointraffic client module makes it easy to implement Cointraffic directly into your website. To get started, simply replace the ``placementId`` with your assigned tracker key. This is dependent on the size required by your account dashboard. For additional information on this module, please contact us at ``support@cointraffic.io``.
+
+# Test Parameters
+```
+  var adUnits = [{
+    code: 'test-ad-div',
+    sizes: [[300, 250]],
```

Comment:
The `sizes` property has been deprecated. Sizes need to be entered by media type. If this is a banner ad unit, the size would be entered like this:

```
mediaTypes: {
                banner: {
                    sizes: [[300, 250]]
                }
            }
```

Which media type(s) does this adapter support?

Reply:
Sorry, updated `sizes` property and added `supportedMediaTypes`